exosphere-shared/src/docker-helper.ls

```diff
@@ -13,10 +14,10 @@ class DockerHelper
     child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}') |> (.to-string!) |> (.split '\n') |> (.includes "#{container-name}/Up")
 
 
-  @ensure-container-is-running = (container-name, image) ->
-    | @container-is-running container-name  =>  return
-    | @container-exists container-name      =>  @start-container container-name
-    | otherwise                             =>  @run-image container-name, image
+  @ensure-container-is-running = (container, done) ~>
+    | @container-is-running container.container-name  =>  return done!
+    | @container-exists container.container-name      =>  @start-container container.container-name; done!
```

Comment:
I think `@start-container` is an async function, and should receive `done` as a callback parameter here.